src/components/dropdown/nz-dropdown-button.component.ts

```diff
@@ -67,15 +68,24 @@ import { NzDropDownComponent } from './nz-dropdown.component';
 })
 
 export class NzDropDownButtonComponent extends NzDropDownComponent implements OnInit, OnDestroy, AfterViewInit {
-  @Input() nzDisable = false;
```

Comment:
Here is a breaking bug fix, using `nzDisable` instead of `nzDisabled` is just a typo, but should be mentioned in change log.